OData/src/System.Web.OData/OData/EntityInstanceContext.cs

```diff
@@ -177,7 +177,7 @@ public bool SkipExpensiveAvailabilityChecks
         /// </summary>
         /// <param name="propertyName">The name of the property to get.</param>
         /// <returns>The value of the property if present.</returns>
-        internal object GetPropertyValue(string propertyName)
+        public virtual object GetPropertyValue(string propertyName)
```

Comment:
Only way I can Moq it.  The serializer needs to check for a null value at a new spot which was breaking UTs that didn't mock this out.